src/pybind/mgr/dashboard/frontend/src/app/shared/services/error-handling.service.ts

```diff
@@ -0,0 +1,18 @@
+import { ErrorHandler, Injectable, Injector } from '@angular/core';
+import { LoggingService } from '../api/logging.service';
+
+@Injectable()
+export class ErrorHandling implements ErrorHandler {
```

Comment:
I suggest to use a other class name, maybe ``JsErrorHandler`` or ``LoggingErrorHandler``.

Reply:
@votdev I've renames the service to `JsErrorHandler`